### PR TITLE
documentation: add LCD panel to OCF JS Servers README.md

### LINKED ISSUE #56: Lack of connection description for Grove LCD RGB
This [guide](https://github.com/01org/SmartHome-Demo/tree/master/ocf-servers/js-servers#setting-up-the-smart-solar-panel) describes how to setup smart solar panel. List all hardware components required and how to setup them, except the connection description for Grove LCD RGB.

### PR DESCRIPTION
**Pull request by gvancuts**
Add instructions to the README.md file (in ocf-servers/js-servers)
on how to connect the Grove LCD RGB panel for the Smart Solar Panel
device.

Fixes #56

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

File: ocf-servers/js-servers/README.md [hunk 1]
```diff
@@ -44,21 +44,25 @@ The connector listed below was derived from the corresponding `*.js` files. Plea
...
+* 1x [Grove LCD RGB panel]
 
 ### Setting up the Solar Panel HW components
 1. Mount the [Solar Panel] and [Line Actuator] as shown in the following pictures (the [Actuator Control Board] is not visible in those pictures, refer to point 2 below for more details on how to connect all three components together):
 ![solar-panel-1](./.pics/solar-panel-1.png)
 ![solar-panel-2](./.pics/solar-panel-2.png)
 ![solar-panel-3](./.pics/solar-panel-3.png)
 2. Connect the [Line Actuator] to the [Actuator Control Board] as follows: ![control-to-actuator] (./.pics/control-to-actuator.png)
-3. Connect the [Actuator Control Board] to the Edison using the `RC` connector as follows: ![control-to-edison] (./.pics/control-to-edison.png) and following the table below:
+3. Connect the [Actuator Control Board] to the [Intel® Edison Board for Arduino] using the `RC` connector as follows: ![control-to-edison] (./.pics/control-to-edison.png) and following the table below:
```
Comment (ttzeng) [inline ocf-servers/js-servers/README.md]: Should be "[Intel Edison **Kit** for Arduino](https://www.google.com.tw/url?sa=t&rct=j&q=&esrc=s&source=web&cd=9&cad=rja&uact=8&ved=0ahUKEwj3krPUx63QAhVCKJQKHQrODkIQFghfMAg&url=http%3A%2F%2Fdownload.intel.com%2Fsupport%2Fedison%2Fsb%2Fedisonarduino_hg_331191007.pdf&usg=AFQjCNHdBAyldSkL9XnZ4vF-XEh7uxZ1wg&sig2=bKqXmlh2pXAKhVvCiayDcQ)"

File: ocf-servers/js-servers/README.md [hunk 2]
```diff
@@ -44,21 +44,25 @@ The connector listed below was derived from the corresponding `*.js` files. Plea
...
-| **-** | `GND` |
-| **+** | `5V` |
-| **RC** | `Digital PWM pin 3` |
+  | Actuator | Edison |
+  |:---:|:---:|
+  | **-** | `GND` |
+  | **+** | `5V` |
+  | **RC** | `Digital PWM pin 3` |
+
+4. Connect the [Grove LCD RGB panel] to any `I2C` connector on the [Grove Base Shield] (https://www.seeedstudio.com/base-shield-v13-p-1378.html?cPath=132_134) that is installed on your [Intel® Edison Board for Arduino] as highlighted in the green circle on this picture:
```
Comment (ttzeng) [inline ocf-servers/js-servers/README.md]: s/Board/kit

Comment (ttzeng) [inline ocf-servers/js-servers/README.md]: The tiny DIP switch on the Grove base shield selects either +5V or +3.3V to the Vcc pins of all sockets on the base shield, it's required to switch to +5V for driving the Grove RGB LCD backlight module.

File: ocf-servers/js-servers/README.md (hunk 1 above)
Comment (gvancuts) [inline ocf-servers/js-servers/README.md]: Hi @ttzeng , I have seen both being used and I went with the one that is used on the main link for the board (and the link that I use in this document): http://www.intel.com/content/www/us/en/support/boards-and-kits/intel-edison-boards/intel-edison-board-for-arduino.html

**Comment by gvancuts** [comment]
I have pushed an updated patch. Can you please take a look?

**Comment by ttzeng** [comment]
LGTM

**Comment by wanghongjuan** [comment]
Hi, @gvancuts It works for me. Thanks.

**Comment by gvancuts** [comment]
Thank you for your review, merging.

**Pull request merged**